### PR TITLE
Flav/content block wrapper

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR refactors `RenderMessageMarkdown` to use `ContentBlockWrapper` and extends support for CSV files. The aim is to provide an affordable solution for generating proper CSV files by allowing users to download them directly from the markdown block.

Additionally, `ContentBlockWrapper` has been enhanced to support multiple content types in the clipboard, offering greater flexibility.

image

### CODE REVIEW

File: front/components/misc/ContentBlockWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import {
```
Comment (flvndvd) [inline front/components/misc/ContentBlockWrapper.tsx:1]: Moved from `CodeBlockBanner`.

File: front/components/actions/dust_app_run/DustAppRunActionDetails.tsx [hunk 2]
```diff
@@ -6,8 +6,8 @@ import { useMemo } from "react";
 import { ActionDetailsWrapper } from "@app/components/actions/ActionDetailsWrapper";
```
Comment (flvndvd) [inline front/components/actions/dust_app_run/DustAppRunActionDetails.tsx:6]: Better to review with hidden whitespaces ([link](https://github.com/dust-tt/dust/pull/6177/files?diff=split&w=1)).

File: front/components/assistant/RenderMessageMarkdown.tsx [hunk 3]
```diff
@@ -493,16 +453,20 @@ function PreBlock({
...
-          type: "text/plain",
-        }),
-      })
-    );
-  };
+  const text = validChildrenContent || fallbackData || "";
+  const language = detectLanguage(children);
+
+  // If the output file is a CSV let the user download it.
+  const getContentToDownload: GetContentToDownloadFunction | undefined =
```
Comment (philipperolet) [inline front/components/assistant/RenderMessageMarkdown.tsx:460]: why not let them download other kinds of content?

**Review by philipperolet — APPROVED** [review]
LGTM :pray: Left a com, assumed you also tested it with code to check there's no regression

File: front/components/assistant/RenderMessageMarkdown.tsx (hunk 3 above)
Comment (flvndvd) [reply front/components/assistant/RenderMessageMarkdown.tsx:460]: This time will come 🙃

**Pull request merged**